CHANGELOG.md

```diff
@@ -4,6 +4,176 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.11.0] - 2023-02-21
+
+## 🚀 Features
+
+### Support for UUID and Unix timestamp functions in Rhai ([PR #2617](https://github.com/apollographql/router/pull/2617))
+
+When building Rhai scripts, it's often that you'll need to add headers that either uniquely identify a request, or append timestamp information for processing information later, such as crafting a trace header or otherwise.
+
+While the default `timestamp()` and similar functions (e.g. `apollo_start`) can be used for a somewhat similar function, it also isn't able to be translated into an epoch. As a result, it doesn't acutely address the asks we've heard from users.
+
+This adds a `uuid_v4()` and `unix_now()` function to obtain a UUID and Unix timestamp, respectively.
+
+By [@lleadbet](https://github.com/lleadbet) in https://github.com/apollographql/router/pull/2617
+
+### Show option to "Include Cookies" in Sandbox
+
+This adds default support for using the "Include Cookies" toggle when working with Embedded Sandbox.
+
+By [@esilverm](https://github.com/esilverm) in https://github.com/apollographql/router/pull/2553
+
+### Add a metric to track the cache size ([Issue #2522](https://github.com/apollographql/router/issues/2522))
+
+We've introduced a new `apollo_router_cache_size` metric that reports the current size of in-memory caches.  Like [other metrics](https://www.apollographql.com/docs/router/configuration/metrics), it is available via OpenTelemetry Metrics including Prometheus scraping.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/2607
+
+### Add a rhai global variable resolver and populate it ([Issue #2628](https://github.com/apollographql/router/issues/2628))
+
+Rhai doesn't place _constants_ into the global scope, instead requiring that functions be pure. We only became aware of this limitation this week, mainly as a consequence of people having difficulties with the experimental `APOLLO_AUTHENTICATION_JWT_CLAIMS` constant.
+
+Our fix is to introduce a new global [variable resolver](https://rhai.rs/book/engine/var.html) and populate it with a `Router` global constant. It currently has three members:
+
+ - `APOLLO_START -> should be used in place of `apollo_start`
+ - `APOLLO_SDL -> should be used in place of `apollo_sdl`
+ - `APOLLO_AUTHENTICATION_JWT_CLAIMS`
+
+You access a member of this variable as follows:
+
+```
+   let my_var = Router.APOLLO_SDL;
+```
+
+We are removing the _experimental_ `APOLLO_AUTHENTICATION_JWT_CLAIMS` constant, but we will **retain the existing non-experimental constants** for purposes of backwards compatibility.
+
+
+We recommend that you shift to the new global constants since we will remove them in a major breaking change release in the future.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/2627
```

Comment:
```suggestion
### Add a rhai global variable resolver and populate it ([Issue #2628](https://github.com/apollographql/router/issues/2628))

Rhai scripts cannot access Rust global constants by default, making cross plugin communication via `Context` difficult.

This change introduces a new global [variable resolver](https://rhai.rs/book/engine/var.html) populates with a `Router` global constant. It currently has three members:

 - `APOLLO_START -> should be used in place of `apollo_start`
 - `APOLLO_SDL -> should be used in place of `apollo_sdl`
 - `APOLLO_AUTHENTICATION_JWT_CLAIMS`

You access a member of this variable as follows:

```
   let my_var = Router.APOLLO_SDL;
```

We are removing the _experimental_ `APOLLO_AUTHENTICATION_JWT_CLAIMS` constant, but we will **retain the existing non-experimental constants** for purposes of backwards compatibility.

We recommend that you shift to the new global constants since we will remove them in a major breaking change release in the future.

By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/2627
```